src/pages/account/logins/index.tsx

```diff
@@ -28,8 +26,25 @@ type LoginsProps = {
 };
 
 const Logins: React.FC<LoginsProps> = ({ sortedIdentities }) => {
-  const [addEmail, setAddEmail] = React.useState<boolean>(false);
-  const [addPhone, setAddPhone] = React.useState<boolean>(false);
+  const [hideSecondaryEmails, setHideSecondaryEmails] = React.useState<boolean>(
+    true,
+  );
+  const [hideSecondaryPhones, setHideSecondaryPhones] = React.useState<boolean>(
+    true,
+  );
+
+  let emailList;
+  let phoneList;
+
+  hideSecondaryEmails
+    ? (emailList = sortedIdentities.emailIdentities.filter((id) => id.primary))
```

Comment:
Using `id` is misleading in this context.
```suggestion
    ? (emailList = sortedIdentities.emailIdentities.filter((identity) => identity.primary))
```

Reply:
OK fixed 0fe5a7d